packages/modules/product/src/repositories/product-category.ts

```diff
@@ -51,19 +51,27 @@ export class ProductCategoryRepository extends DALUtils.MikroOrmBaseTreeReposito
     }
 
     const shouldExpandParent =
-      familyOptions.includeAncestorsTree || populate.includes("parent_category") || fields.some(field => field.startsWith('parent_category'))
+      familyOptions.includeAncestorsTree ||
+      populate.includes("parent_category") ||
+      fields.some((field) => field.startsWith("parent_category"))
 
     if (shouldExpandParent) {
-      populate.indexOf("parent_category") === -1 &&
-        populate.push("parent_category")
+      // Since the tree is manually built, we dont want to get the relations
+      const relationIndex = populate.indexOf("parent_category")
+      relationIndex !== -1 && populate.splice(relationIndex, 1)
+      familyOptions.includeAncestorsTree ??= true
```

Comment:
We cannot assume that expanding `parent_category` is the same as including ancestors. `include_ancestors` will expand all parent categories, while as only passing `parent_category` is only the immediate parent.

Reply:
Same goes for descendants

Reply:
ok, got it, I ll change that 👍